keyboards/3.0/fr-t-k0-optimise.xml

```diff
@@ -0,0 +1,230 @@
+		<key id="mark-stroke" output="\m{stroke}" />
+		<key id="mark-submacron" output="\m{submacron}" />
+		<key id="mark-tilde" output="\m{tilde}" />
+		<key id="mark-umlaut" output="\m{umlaut}" />
+
+
+		<!-- extra keys -->
+		<key id="e-grave" output="è" />
+		<key id="e-acute" output="é" />
+		<key id="c-cedilla" output="ç" />
+		<key id="a-acute" output="á" />
+		<key id="a-grave" output="à" />
+		<key id="e-caret" output="ê" />
+
+		<!-- extra symbols -->
+		<key id="bullet" output="•" />
+		<key id="umlaut" output="¨" />
+		<key id="sub-2" output="₂" />
+		<key id="super-2" output="²" longPressKeyIds="sub-2" />
+		<key id="en-dash" output="–" />
+		<key id="plus-minus" output="±" />
+		<key id="vulgar-half" output="½" />
+		<key id="vulgar-quarter" output="¼" />
+		<key id="ellipsis" output="…" />
+		<key id="open-apos" output="‘" />
+		<key id="close-apos" output="’" />
+		<key id="open-g" output="«" />
+		<key id="close-g" output="»" />
+
+		<key id="theta" output="θ" /> <!-- "for Romani" -->
+		<key id="Theta" output="ϴ" /> <!-- "for Romani" -->
+		<key id="tm" output="™" />
+		<key id="open-double" output="“" />
+		<key id="close-double" output="”" />
+		<key id="oe" output="œ" />
+		<key id="Oe" output="Œ" />
+		<key id="registered" output="®" />
+		<key id="minus-sign" output="−" />
+		<key id="ss" output="ß" />
+		<key id="ininity" output="∞" />
+		<key id="division" output="÷" />
+		<key id="less-equal" output="≤" />
+		<key id="greater-equal" output="≥" />
+		<key id="copy" output="©" />
+		<key id="inverse-question" output="¿" />
+		<key id="inverse-bang" output="¡" />
+		<key id="middle-dot" output="•" />
+		<key id="A-grave" output="À" />
+		<key id="E-acute" output="É" />
+		<key id="E-caret" output="Ê" />
+		<key id="em-dash" output="—" />
+		<key id="open-single" output="’" />
+		<key id="close-single" output="’" />
+		<key id="u-grave" output="Ù" />
+		<key id="radix" output="√" />
+		<key id="C-cedilla" output="Ç" />
+		<key id="not-equal" output="≠" />
+        <key id="open-angle-quote" output="‹" />
+        <key id="close-angle-quote" output="›" />
+
+		<key id="infinity" output="∞" />
+		<key id="E-grave" output="È" />
+		<key id="U-grave" output="Ù" />
+		<key id="nb-hyphen" output="‑" />
+		<key id="multiplication" output="×" />
+		<key id="ezh" output="ʒ" />
+		<key id="Ezh" output="Ʒ" />
+		<key id="ae" output="æ" />
+		<key id="Ae" output="Æ" />
+		<key id="permille" output="‰" />
+		<key id="dbl-dagger" output="‡" />
+		<key id="dagger" output="†" />
+		<key id="asym-equal" output="≃" />
+		<key id="Ss" output="ẞ" />
+        <!-- interestingly, this key has no output by itself, but is U+00A4 when preceded by the currency mark. -->
+        <key id="altshifte" output="\m{altshifte}"/>
+	</keys>
+
+	<layers formId="iso">
+		<layer modifiers="none">
+			<row
+				keys="at a-grave e-acute e-grave e-caret open-paren close-paren open-apos close-apos open-g close-g apos mark-caret" />
```

Comment:
The circumflex dead key (listed here as `mark-caret`) must also grant access to superscript numerals (0 to 9).

Reply:
I missed that, thanks.